internal/http/services/owncloud/ocdav/propfind/propfind.go

```diff
@@ -708,6 +708,14 @@ func mdToPropResponse(ctx context.Context, pf *XML, md *provider.ResourceInfo, p
 		Propstat: []PropstatXML{},
 	}
 
+	if status := utils.ReadPlainFromOpaque(md.Opaque, "status"); status == "processing" {
+		response.Propstat = append(response.Propstat, PropstatXML{
+			Status: "HTTP/1.1 425 TOO EARLY", // TODO: use proper status code
```

Comment:
Yeah, 425 is about replay attacks and the `Early-Data` header.

IMO [202 Accepted] would be the correct Status for a resource that is still being uploaded:
> The *202 (Accepted)* status code indicates that the request has been accepted for processing, but the processing has not been completed. The request might or might not eventually be acted upon, as it might be disallowed when processing actually takes place. There is no facility in HTTP for re-sending a status code from an asynchronous operation.

But existing desktop clients might try to download files with a 200 status ... @TheOneRing what would desktop clients do with status 202? ignore the file? would be great for now...

Reply:
I'm pretty sure to guess what a 423 will cause.
For everything indicating ok I'd expect a download to be triggered. If you need more info I'd need to dig deeper.

Reply:
As discussed in cross platform meeting we will stick with the 425 status code for now. We can revisit that decision later.